examples/manifestworkclient/client/README.md

```diff
@@ -0,0 +1,69 @@
+# Resource Bundle CURD with Manifestwork Client
+
+## Preparation
+
+1. Enable gRPC server by passing `--enable-grpc-server=true` to the maestro server start command:
```

Comment:
Do we need to enable gPRC server manually? I think we should make it enabled by default. WDYT?

Reply:
yes, we can do it.
if that mean delete the `--enable-grpc-server` flag? my concern is upgrade issue.

Reply:
done